demo/main.js

```diff
@@ -111,7 +111,10 @@ shakaDemo.Main = class {
         return;
       }
 
-      this.onError_(/** @type {!shaka.util.Error} */ (errorEvent.error));
+      const shakaError = /** @type {!shaka.util.Error} */ (errorEvent.error);
+      if (shakaError) {
```

Comment:
What's the logic for ignoring these?  Do they have a `message` field?  I think if we want to ignore uncaught errors, we should be careful.